INSTALL.md

```diff
@@ -103,3 +111,24 @@ make py-deps
 ```
 
 See [GitHub Actions workflow](.github/workflows/main.yml) for other usages of the Makefile.
+
+## Tests
+
+To test the WebAssembly module we will use the [cypress](https://www.cypress.io/) test framework.
+
+We want to test if visiting the example web page renders the answer `-1.00`.
+
+```{.js file=cypress/integration/example_spec.js}
```

Comment:
```suggestion
```{.js file=cypress/integration/example_spec.js}
   // this snippet will be used as example to test the web page
```

Reply:
Done in cb2400e1413b42a14833570a2cf78d43bc2d4efc